examples/skywalking-tracing/docker-compose.yaml

```diff
@@ -0,0 +1,112 @@
+version: "3.7"
+services:
+
+  # Front envoy.
+  front-envoy:
+    build:
+      context: .
+      dockerfile: Dockerfile-frontenvoy
+    networks:
+      - envoymesh
+    expose:
+      - "8000"
+      - "8001"
+    ports:
+      - "8000:8000"
+      - "8001:8001"
+    depends_on:
+      - skywalking-oap
+  # First service.
+  service1:
+    build:
+      context: ../front-proxy
+      dockerfile: Dockerfile-service
+    volumes:
+      - ./service1-envoy-skywalking.yaml:/etc/service-envoy.yaml
+    networks:
+      - envoymesh
+    environment:
+      - SERVICE_NAME=1
+    expose:
+      - "8000"
+    depends_on:
+      - skywalking-oap
+  # Second service.
+  service2:
+    build:
+      context: ../front-proxy
+      dockerfile: Dockerfile-service
+    volumes:
+      - ./service2-envoy-skywalking.yaml:/etc/service-envoy.yaml
+    networks:
+      - envoymesh
+    environment:
+      - SERVICE_NAME=2
+    expose:
+      - "8000"
+    depends_on:
+      - skywalking-oap
+  # Skywalking components.
+  elasticsearch:
+    image: elasticsearch:7.9.2
+    container_name: elasticsearch
+    restart: always
+    networks:
+      - envoymesh
+    ports:
+      - 9200:9200
+    healthcheck:
+      test: ["CMD-SHELL", "curl --silent --fail localhost:9200/_cluster/health || exit 1"]
```

Comment:
i tend to do this for elasticsearch 
```
curl -s http://localhost:9200/_cluster/health | grep -vq '"status":"red"`
```
i think it doesnt fail even if the cluster isnt ready

Reply:
Actually, I'm not familiar with ES. I copy this docker compose config from [skywalking community‘s examples](https://github.com/apache/skywalking-docker/blob/master/8/8.1.0/compose-es7/docker-compose.yml) and didn't make any changes.

Reply:
feel free to ignore this nit - i have some vague mem of hitting a problem with ES being ready without  checking status